device/iot-device-samples/send-receive-x509-sample/src/main/java/samples/com/microsoft/azure/sdk/iot/SSLContextBuilder.java

```diff
@@ -1,276 +0,0 @@
-/*
```

Comment:
Since this file was just a copy from another sample's SSLContextBuilder, I'm removing it and adding a dependency to the other sample